finalProject/tenSpeckledFrogs.js

```diff
@@ -0,0 +1,30 @@
+/*
+> 3 speckled frogs sat on a log
+> eating some most delicious bugs.
+> One jumped in the pool where its nice and cool,
+> then there were 2 speckled frogs.
+>
+> 2 speckled frogs sat on a log
+> eating some most delicious bugs.
+> One jumped in the pool where its nice and cool,
+> then there was 1 speckled frog.
+>
+> 1 speckled frog sat on a log
+> eating some most delicious bugs.
+> One jumped in the pool where its nice and cool,
+> then there were no more speckled frogs!
+*/
+
+var frogs = 10;
+
+ for (i = frogs; i > 0; i --) {
+   if(i > 2) {
```

Comment:
This is the condition that will be hit the most- typically, the else block is used for that, with the special conditions coming first.